docs/terms.html

```diff
@@ -95,7 +101,7 @@ <h3 class="govuk-heading-s">Version effective from 23rd March 2020.</h2>
         I agree to only upload files types currently supported by Street Manager with a single file size limit of 10MB and a 100MB total limit on file uploads for each individual permit.
       </p>
       <p class="govuk-body">
-        The DfT can change these terms and conditions at any time. We will tell you about any change by email at least 10 calendar days before the change happens. The DfT will specify the date the change will happen in the message you receive. If you do not accept the change, you must notify the DfT and stop using Street Manager.
+        Invoices will be sent quarterly in arrears. Charges will be incurred from the 1st day of the month following the date you signed up to the Street Manager service. If an organisation changes its status from contractor to promoter charges will apply.The DfT can change these terms and conditions at any time. We will tell you about any change by email at least 10 calendar days before the change happens. The DfT will specify the date the change will happen in the message you receive. If you do not accept the change, you must notify the DfT and stop using Street Manager.
```

Comment:
Feels like a comma is needed here 
```suggestion
        Invoices will be sent quarterly in arrears. Charges will be incurred from the 1st day of the month following the date you signed up to the Street Manager service. If an organisation changes its status from contractor to promoter, charges will apply. The DfT can change these terms and conditions at any time. We will tell you about any change by email at least 10 calendar days before the change happens. The DfT will specify the date the change will happen in the message you receive. If you do not accept the change, you must notify the DfT and stop using Street Manager.
```